### DISCUSSION TITLE
Error initialisation

### DISCUSSION THREAD

**Discussion by Morcegolas** [albaintor/homeassistant_electrolux_status#14] [Q&A]
Hello, I'm trying to get this working, but I get this error on the log and no devices show up. 

'NoneType' object is not iterable
Traceback (most recent call last):
  File "/config/custom_components/electrolux_status/__init__.py", line 104, in setup_entities
    for appliance_json in appliances_list:
TypeError: 'NoneType' object is not iterable

My washing machine is AEG Series 7000 LFR7394O4Q ProSteam

**Comment by wgumaa** [comment]
I am getting the same. I have enabled debugging logging but don't know where to find the logs.

**Comment by albaintor** [comment]
Yes I am stuck for now : the server rejects (randomly) the requests that the component sends.

**Reply by Morcegolas** [reply]
Tried now with 1.0.2 and it showed up the machine ;)
